Flyte Scribe/presence.ts

```diff
@@ -0,0 +1,31 @@
+var presence = new Presence({
+    clientId: "707632555612045413"
+  }),
+  presenceData: presenceData = {
+    largeImageKey: "logo"
+  }
```

Comment:
you forgot a `;` at the end, it should fix your deepscan issue.

Reply:
Added the `;` in presence.ts and recompiled in https://github.com/PreMiD/Presences/pull/1511/commits/d0b4847cfff91e1c8b4dfb51a778f5b89868b1cd